custom_components/ramses_cc/binary_sensor.py

```diff
@@ -96,32 +99,25 @@ def entity_factory(broker, device, attr, *, entity_class=None, **kwargs):
             platform.async_register_entity_service(name, schema, f"svc_{name}")
 
 
-class RamsesBinarySensor(RamsesDeviceBase, BinarySensorEntity):
+class RamsesBinarySensor(RamsesSensorBase, BinarySensorEntity):
     """Representation of a generic binary sensor."""
 
     def __init__(
         self,
         broker,  # ramses_cc broker
-        device,  # ramses_rf device
+        rf_entity: RamsesRFEntity,
         state_attr,  # key of attr_dict +/- _ot suffix
         device_class=None,  # attr_dict value
         **kwargs,  # leftover attr_dict values
     ) -> None:
         """Initialize a binary sensor."""
-
-        _LOGGER.info("Found a Binary Sensor for %s: %s", device.id, state_attr)
-
-        super().__init__(
-            broker,
-            device,
-            state_attr,
-            device_class=device_class,
-        )
+        _LOGGER.info("Found %r: %s", rf_entity, state_attr)
```

Comment:
I've shortened these logs and standardised on use of RF entity repr as the logger name already indicates type.

Reply:
OK.